cli/src/main/kotlin/com/meamoria/lexurgy/cli/Application.kt

```diff
@@ -8,6 +8,9 @@ import kotlin.time.DurationUnit
 import kotlin.time.ExperimentalTime
 import kotlin.time.measureTimedValue
 
+class LscUnsupportedDirective() : LscUserError(
+    "Include directives are not supported in load_string requests"
+)
```

Comment:
Not sure this is really necessary, but since there seems to be some distinction between `UserError`s and other exceptions, I didn’t just want to do `throw Exception`.